apps/rehabstod/src/utils/getRekoStatusSickLeaveTimestamp.ts

```diff
@@ -0,0 +1,4 @@
+import { format, isAfter } from 'date-fns'
+
+export const getRekoStatusSickLeaveTimestamp = (endDate: string) =>
```

Comment:
Nice that you made this into a reusable util and added tests to verify the logic.